org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintUiPlugin.java

```diff
@@ -172,6 +181,17 @@ public synchronized void closeSonarConsole() {
     }
   }
 
+  public synchronized ListenerFactory listenerFactory() {
+    if (listenerFactory == null) {
+      listenerFactory = () -> notification -> Display.getDefault().asyncExec(() -> {
+        SonarQubeNotificationPopup popup = new SonarQubeNotificationPopup(Display.getCurrent(), notification);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "popup" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)